biblatex.cfg

```diff
@@ -0,0 +1,20 @@
+%Configure biblatex to fit ISO-690 UTBM's takes
+%Expect biblatex style to be authortitle as iso-authoryear force italic for title
+\DefineBibliographyStrings{french}{% Define words to prefix url and urldate
+  urlseen = {consulté le},
+  urlfrom = {disponible sur},
+}
+
+\DeclareFieldFormat{urldate}{\mkbibparens{\bibstring{urlseen}\space#1}}% Add parenthethis insted of crochet for the consult date and add urlseen as prefix
+
+\DeclareFieldFormat{url}{\bibstring{urlfrom}\addcolon\space\url{#1}}% Format the URL to be prefixed by urlfrom
+
+\DeclareFieldFormat[online]{titleaddon}{\text{In : }\textit{#1}} % Italicize titleaddon aka website name and add In : prefix
```

Comment:
Ici aussi ça traduit en français ou ça laisse `In :` ?

Reply:
Faut laisser le In :  devant le nom du site

ça donne un truc dans ce style : 

GAULT, Matthew. Canon Tells Customers to Break Its Printer Cartridge DRM Due to Chip Shortage. In : Vice [en ligne]. 11 janvier 2022. Disponible sur : https://www.vice.com/en/article/88g835/canon-tells-customers-to-break-its-printer-cartridge-drm-due-to-chip-shortage (consulté le 04/12/2023)